storybookjs/storybook
# Merge pull request #9492 from storybookjs/9490-fix-source-loader-imports

Issue: #9490 

## What I did

Always insert the preamble code at the beginning of the file. See the repro in #9490 

```
import A1
export A2
import B1
-> (inserted code)
export B2
```

=>

```
-> (inserted code)
import A1
export A2
import B1
export B2
```

## How to test

See new `official-storybook` example, which fails before the change & succeeds after
